docs/DEBUGGING.md

```diff
@@ -451,3 +451,7 @@ Further parsing and diffing the files is possible using tpm2-tools:
 >   Digests:
 [...]
 ```
+
+### TPM Recovery
+
+Using `recovertpm` tool from debug container, it is possible to perform TPM hardware test, key re-generation, volume key extraction and more. Note that the extracted key is in encrypted wire format suitable for a cloud controler, and key extraction is only possible if the device state (current PCR values) match the original state where key was seald.
```

Comment:
typo "seald".

Does recovertpm -h list the set of options?
If so it might make sense to mention that here.

Reply:
yes it does, you mean just mention `recovertpm -h` or the output of `recovertpm -h` ?

Reply:
Just mention it as the way to find out the options.

Reply:
done.